File: polyglot/seis_native/r/seis_native_kernel.R
seis_native_roadmap <- data.frame(
  lane = c("Apple First", "Data AI", "Systems", "Android", "Windows", "Infrastructure"),
  score = c(100, 88, 84, 76, 72, 70),
  primary_language = c("Swift", "Python", "Rust", "Kotlin", "CSharp", "Go"),
  stringsAsFactors = FALSE
)

seis_native_top_lane <- function() {
  seis_native_roadmap[which.max(seis_native_roadmap$score), ]
}

File: polyglot/seis_native/dart/seis_native_kernel.dart
enum NativeLaneName {
  appleFirst,
  dataAI,
  systems,
  android,
  windows,
  infrastructure,
}

class NativeRoadmapItem {
  const NativeRoadmapItem({
    required this.lane,
    required this.score,
    required this.primaryLanguage,
  });

  final NativeLaneName lane;
  final int score;
  final String primaryLanguage;
}

const nativeRoadmap = <NativeRoadmapItem>[
  NativeRoadmapItem(lane: NativeLaneName.appleFirst, score: 100, primaryLanguage: 'Swift'),
  NativeRoadmapItem(lane: NativeLaneName.dataAI, score: 88, primaryLanguage: 'Python'),
  NativeRoadmapItem(lane: NativeLaneName.systems, score: 84, primaryLanguage: 'Rust'),
  NativeRoadmapItem(lane: NativeLaneName.android, score: 76, primaryLanguage: 'Kotlin'),
  NativeRoadmapItem(lane: NativeLaneName.windows, score: 72, primaryLanguage: 'CSharp'),
  NativeRoadmapItem(lane: NativeLaneName.infrastructure, score: 70, primaryLanguage: 'Go'),
];

NativeRoadmapItem topLane() => nativeRoadmap.first;
